fiscalsim_us/tests/policy/baseline/gov/states/mn/tax/income/mn_amt.yaml

```diff
@@ -6,7 +6,7 @@
     mn_basic_tax: 0
     state_code: MN
   output:
-    mn_amt: 675
+    mn_amt: 4_708.125

```

Comment:
@mitchellpound. Two things.
1. We should round these to the nearest dollar. You can do this in the variable `.py` files.
2. These amounts are significantly different from their original values. Can you explain why?

Reply:
@rickecon. 
- I added a rounding rule in the `mn_amt.py` file. 
- The old test number was blatantly wrong. After the MN amt taxable income is computed, the amt is a flat tax of 6.75%, so you can just multiply the two numbers to get the amt. The old test output would imply an amt tax rate of <1%

Reply:
Hello @mitchellpound-

Just FYI, I think the original value might actually be correct: there's a few more steps in between the taxable income and the tax amount that involve thresholds and rates that are not the 6.75%. 

If after transferring the PolicyEngine code in here you do find other places where you think there is an error, in open-source spirit please do feel very free to file an issue to let us know! As an analogy, whenever we find a bug or speedup opportunity in PolicyEngine Core whose fix would be compatible with OpenFisca Core, we make sure to file an issue to let their team know about it.

Best,

Nikhil

Reply:
@nikhilwoodruff, you are so right!! Thank you for pointing that out. 

I checked, and in my version of the MN tax code, I left that logic for the other steps inside the `mn_amt_taxable_income`  since it's referred to as the AMT standard deduction by the MN house research committee. However, the PolicyEngine code has that logic inside the `mn_amt` variable. This means that the test should be correct on the PolicyEngine side, but this updated version is correct for the tax logic inside fiscalsim. 

I'll keep an eye out for other 'bugs' as I go through the code base :thumbsup: